_posts/2019-01-10-Flutter.md

```diff
@@ -0,0 +1,185 @@
+---
+layout: post
+authors: [dimitri_de_kerf]
+title: 'Flutter: Hybrid apps for mobile & beyond.'
+image: /img/2018-12-17-Flutter/Flutter.jpg
+tags: [Android,Flutter,Hybrid,iOS,Mobile]
+category: Development
+comments: true
+---
+
+## Intro
+
+Mobile development has always intrigued me.
+Bringing data to life by visualizing it with different components and being able to carry your app along inside your pocket has something special.
+I still remember the excitement I felt when working on my first mobile app in university,
+even with the lack of good resources back then and the sluggish emulator which was available at that time.
+
+The mobile world has kept expanding with big improvements in resources, frameworks, tools and designs.
+From the first iPhone to having your refrigerator running your favorite apps, more and more possibilities and challenges have become available to keep you going.
+Nowadays, there are different paths that you can follow to create those apps.
+Native, hybrid or web apps, it all depends on what you want to achieve and how many resources are available.
+Each has its benefits and pitfalls, which doesn’t make the decision easy which path to follow.
+Do you want to give your audience the best native experience with great performance?
+Or does a hybrid app suffices where you might compromise in speed and in look & feel of a native app?
+Flutter might be the answer, by providing you the best parts of both worlds.
+
+## Beautiful native apps in record time
+
+>Flutter allows you to build beautiful native apps on iOS and Android from a single codebase.
+
+<span class="image left"><img alt="Flutter" src="/img/2018-12-17-Flutter/flutter_logo.jpeg"></span>
+
+A promising statement which is presented to you when you browse to [Flutter.io](https://flutter.io/){:target="_blank" rel="noopener noreferrer"}.
+A statement that explains perfectly what Flutter is in fact.
+Flutter is an open source mobile UI framework made by Google.
+With Flutter, you can build beautiful apps that run at native speed.
+Given the single codebase, you don’t have to develop the same app twice for both iOS and Android.
+Flutter is even the first-class citizen for Fuchsia, an upcoming mobile OS that is currently being developed by Google.
```

Comment:
Not much has been known yet about Fuchsia since it's still in development and Google wants to keep it private. Google has recently removed all of the UI part, so the previous renderers of the OS aren't valid anymore. My guess is that they will adopt the Material Design and that Fuchsia will look more like Android than iOS since Android apps will be compatible with Fuchsia.

Reply:
I would mention that this is undecided yet